tests/integration_tests/build/test_binary_size.py

```diff
@@ -13,9 +13,9 @@
 
 SIZES_DICT = {
     "x86_64": {
-        "FC_BINARY_SIZE_TARGET": 1955168,
+        "FC_BINARY_SIZE_TARGET": 2059824,
         "JAILER_BINARY_SIZE_TARGET": 1439512,
-        "FC_BINARY_SIZE_LIMIT": 2052927,
+        "FC_BINARY_SIZE_LIMIT": 2100000,
```

Comment:
Do we understand this increase? Can you detail it? just looking on the code a 2% increase doesn't make sense to me

Reply:
Given the delta of 5%, the increase is not based only on this PR. The size has likely increased over time, and this PR was unlucky enough to go overboard the 5%

Reply:
I saw this exact same issue on this PR #2223 , for ARM

Reply:
Ok.